browser.go

```diff
@@ -310,6 +310,11 @@ func (b *Browser) run(ctx context.Context) {
 	}
 }
 
+// Pid returns the browser process ID
+func (b *Browser) Pid() int {
```

Comment:
According to [Go Code Review requirement](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
```suggestion
func (b *Browser) PID() int {
```

Reply:
I'm sorry, but I think it's better to make it consistant with the standard package, which is `Pid`. Or even better, call it `Getpid` (because this is a method instead of a field)? What do you think? @dmytro-vovk